ADOPTERS.md

```diff
@@ -23,3 +23,7 @@
 * [Devino Telecom](https://devinotele.com) uses CoreDNS with default configuration for Kubernetes for its Lab and its Production.
 * [Yandex Money](https://money.yandex.ru) uses CoreDNS in Lab and Production, using default configuration for Kubernetes.
 * [AdGuard](https://adguard.com/) uses CoreDNS in [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome) and, therefore, in production public AdGuard DNS servers.
+* [Skyscanner](https://www.skyscanner.net) uses CoreDNS within Kubernetes in production with the configuration tuned to use the Autopath plugin.
+* [Zinza Technology](http://zinza.com.vn) uses CoreDNS within Kubernetes in production, with standard configuration.
+* [Hualala](https://hualala.com)  uses CoreDNS in Kubernetes using default configuration, in its Lab. Expected to be in production soon.
```

Comment:
this website looks shady - https using a bad cert, and http://www.hualala.com/ never loads...